src/oscar/templates/oscar/search/results.html

```diff
@@ -42,13 +42,12 @@
 
 {% block content %}
     <form method="get" action="." class="form-horizontal">
-        {# Render other search params in a hidden block #}
-        <div style="display:none">
-            {% for value in selected_facets %}
-                <input name="selected_facets" value="{{ value }}" />
-            {% endfor %}
-            {{ search_form.q }}
-        </div>
+        {# Render other search params as hidden inputs #}
+        {% for value in selected_facets %}
+            <input type="hidden" name="selected_facets" value="{{ value }}" />
+        {% endfor %}
+        <input type="hidden" name="q" value="{{ search_form.q.value }}" />
```

Comment:
This would still insert an `id` attribute that conflicts with the navbar search form?